### ISSUE TITLE
AG WG Review: 3.1.5 Reading Level (Level AAA) WCAG2ICT Guidance

### ISSUE THREAD

**Issue by pday1** [w3c/wcag2ict#944]
## Summary
Review of proposed WCAG2ICT guidance for the Level AAA WCAG SC: 3.1.5 Reading Level, as incorporated into the Editor's draft. 

## Maturity Level
Refining

## Action Needed
* Review content (See Content for Review section below for details.)
* Use the thumbs up and thumbs down icons to indicate agreement or disagreement with various comments. Do not add comments to simply support other comments. 
* Note any suggested edits for improvement and/or reasoning either in a comment on this issue or by creating a PR on the WCAG2ICT document.  
* Level A and AA SCs can be found in the [comments-by-guideline-and-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-by-guideline-and-success-criterion.md) file.
* Level AAA SCs can be found in the [comments-on-level-AAA-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-on-level-AAA-success-criteria.md)
----

## Background information 

Once finalized, this new WCAG2ICT Note will build on the previous version by also covering WCAG 2.2 level AAA success criteria (currently only level A and AA criteria are covered.  This is important as different jurisdictions are using different versions of WCAG for the accessibility of information and communication technology. 

## Content for Review

* Section [Applying SC 3.1.5 Reading Level to non-web documents and non-web software](https://w3c.github.io/wcag2ict/#applying-3-1-5-reading-level-to-non-web-documents-and-non-web-software).  

## Reference Material
In case you need them, links to various resources:

* [3.1.5 Reading Level](https://www.w3.org/TR/WCAG22/#reading-level) SC text in WCAG 2.2
* [Intent section of Understanding 3.1.5 Reading Level](https://www.w3.org/WAI/WCAG22/Understanding/reading-level.html#intent)

**Comment by maryjom** [comment]
This is another success criterion that seems a bit ambiguous in WCAG when you're applying it to non-web software. Would you actually apply this success criterion to the text labels on individual user interface components which are typically only 1-3 words long? Or would application of this SC be limited to longer prose (sentences, paragraphs, etc.) that would appear as hints, messages and alerts, help, etc.? 

This is also something that should be addressed in WCAG (at least in WCAG 3). The examples provided in the understanding 3.1.5 are all regarding prose - documents, research papers, and course materials and none about a user interface and the interactive components within it such as what would appear in a web-based application. The e-learning application example is about the content being presented (a learning module) and not the user interface of the application itself.

**Comment by bruce-usab** [comment]
> Would you actually apply this success criterion to the text labels on individual user interface components which are typically only 1-3 words long?

No, because the SC applies only after (emphasis added)  “after removal of proper names and **titles**” and text labels on UIC are a kind of title.  I think a better way to address this is an update to Understanding, since it‘s an issue for Web apps and not specific to software.
